### PR TITLE
libreoffice: update to 6.2.0.3.

### PR DESCRIPTION
**Pull request by Johnnynator**
ToDo:
- [x] test Runtime
- [ ] Java splitting ( #8486 ) if possible
- [x] Update some distfiles, The configure script is currently fetching some that are already fetched by the template.
- [ ] Splitting/Removing Dictionaries (#8756)

### CODE REVIEW

**Comment by Johnnynator** [comment]
Test repo if anyone feels like testing it: https://ablan.de/void/libreoffice/

File: srcpkgs/libreoffice/template [hunk 1]
```diff
@@ -533,6 +536,7 @@ do_install() {
 libreoffice-common_package() {
 	short_desc+=" - Common files"
 	depends="hunspell hyphen mythes openldap"
+	skiprdeps=/usr/lib/libreoffice/program/libofficebean.so
```
Comment (Johnnynator) [inline srcpkgs/libreoffice/template]: @pullmoll What is you opinion on just ignoring libraries that are required by `libofficebean.so`, LibreOffice afaik only tries to load it when Java is installed. This should resolve #8486 to a degree.

Comment (Johnnynator) [reply srcpkgs/libreoffice/template]: The diff of xbps-query between with and without skiprdeps
```diff
--- /proc/self/fd/12	2019-02-17 00:05:38.794816755 +0100
+++ /proc/self/fd/14	2019-02-17 00:05:38.795816749 +0100
@@ -1,13 +1,10 @@
 architecture: x86_64
-automatic-install: yes
 build-date: 2019-02-16 23:11 CET
 build-options: java 
 filename-sha256: 9608a303bb3c595ea36b7f1b4d2e8e4a87657f36267db7ec60d1f1568807cf28
 filename-size: 143MB
 homepage: https://www.libreoffice.org/
-install-date: 2019-02-17 00:02 CET
 installed_size: 599MB
-install-script: 1118B
 	ld-linux-x86-64.so.2
 	libboost_date_time.so.1.69.0
 	libboost_locale.so.1.69.0
@@ -93,11 +90,8 @@
 	libz.so.1
 license: GPL-3.0-or-later
 maintainer: Jürgen Buchmüller <pullmoll@t-online.de>
-metafile-sha256: 6725b8a118fb8c8530756aaf2ba2cea153c8d16b1b3f34c01fb9f84562b36db6
 pkgver: libreoffice-common-6.2.0.3_1
-remove-script: 1115B
 repository: https://ablan.de/void/libreoffice/
 shlib-provides:
 shlib-requires:
 short_desc: Productivity suite - Common files
-state: installed
```

Comment (pullmoll) [reply srcpkgs/libreoffice/template]: I think and hope it's ok to just ignore the dependency and we should give it a try.
If we get bug reports for 6.2.0.3 we know where to look :-)

**Pull request merged**